web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/FlyktningVilkårRoutes.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.su.se.bakover.web.routes.søknadsbehandling
+
+import arrow.core.getOrHandle
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import no.nav.su.se.bakover.domain.Brukerrolle
+import no.nav.su.se.bakover.domain.satser.SatsFactory
+import no.nav.su.se.bakover.service.søknadsbehandling.SøknadsbehandlingService
+import no.nav.su.se.bakover.service.vilkår.LeggTilFlyktningVilkårRequest
+import no.nav.su.se.bakover.web.Resultat
+import no.nav.su.se.bakover.web.features.authorize
+import no.nav.su.se.bakover.web.routes.revurdering.json
+import no.nav.su.se.bakover.web.routes.vilkår.flyktning.LeggTilVurderingsperiodeFlyktningVilkårJson
+import no.nav.su.se.bakover.web.routes.vilkår.flyktning.tilResultat
+import no.nav.su.se.bakover.web.routes.vilkår.flyktning.toDomain
+import no.nav.su.se.bakover.web.svar
+import no.nav.su.se.bakover.web.withBehandlingId
+import no.nav.su.se.bakover.web.withBody
+
+internal fun Route.flyktningVilkårRoutes(
+    søknadsbehandlingService: SøknadsbehandlingService,
+    satsFactory: SatsFactory,
+) {
+    post("$behandlingPath/{behandlingId}/flyktning") {
```

Comment:
burde denne vært en `patch`?

Reply:
Har vel bare brukt post her uten å reflektere så veldig mye over hva somer er riktig - hvis vi skulle tatt den diskusjonen ville jeg kanskje foreslått `put` som det mest riktige for vår bruk.